packages/react-app/src/components/field-input/address-field-input.tsx

```diff
@@ -62,25 +76,29 @@ export function AddressFieldInput({
     );
   const loadableContent = isEdit ? (
     <WrapperStyled>
-      <TextInputStyled id={id} value={value} onChange={onChange} />
+      <DivStyled>
+        <TextInputStyled id={id} value={value} onChange={onChange} />
+      </DivStyled>
       <EthIdenticonStyled address={value} scale={1.6} />
     </WrapperStyled>
   ) : (
     <AddressField address={value} />
   );
   return label ? (
     <>
-      <Field
+      <FieldHeaderStyled
         label={
           <>
             <span title={tooltip}>{label}</span>
-            {tooltip && <IconTooltip tooltip={tooltip} tooltipDetail={tooltipDetail} />}
+            <LabelStyled>
```

Comment:
Nitpicking: It should be TooltipStyled and should be wrapped in IconTooltip instead of having duplicate code & styles in each field ? (Suggestion purpose only)